bld/unit_testers/build-namelist_test.pl

```diff
@@ -1449,7 +1449,24 @@ sub cat_and_create_namelistinfile {
 print "==================================================\n";
 
 # Check for crop resolutions
-my @crop_res = ( "1x1_numaIA", "1x1_smallvilleIA", "4x5", "10x15", "0.9x1.25", "1.9x2.5", "ne3np4.pg3", "ne30np4", "ne30np4.pg3", "C96", "mpasa120" );
+my $crop1850_res = "1x1_smallvilleIA";
+$options = "-bgc bgc -crop -res $crop1850_res -use_case 1850_control -envxml_dir .";
+&make_env_run();
+eval{ system( "$bldnml $options  > $tempfile 2>&1 " ); };
+is( $@, '', "$options" );
+$cfiles->checkfilesexist( "$options", $mode );
+$cfiles->shownmldiff( "default", "standard" );
+if ( defined($opts{'compare'}) ) {
+   $cfiles->doNOTdodiffonfile( "$tempfile", "$options", $mode );
+   $cfiles->dodiffonfile( "$real_par_file", "$options", $mode );
+   $cfiles->comparefiles( "$options", $mode, $opts{'compare'} );
+}
+if ( defined($opts{'generate'}) ) {
+   $cfiles->copyfiles( "$options", $mode );
+}
+&cleanup();
+
+my @crop_res = ( "1x1_numaIA", "4x5", "10x15", "0.9x1.25", "1.9x2.5", "ne3np4.pg3", "ne30np4", "ne30np4.pg3", "C96", "mpasa120" );
```

Comment:
Explaining the mods in this file:
- Moved smallville out of the 2000 cases because it should be looking for an 1850 file.